src/Compilers/CSharp/Portable/Symbols/Attributes/WellKnownAttributeData/TypeWellKnownAttributeData.cs

```diff
@@ -32,5 +32,23 @@ public NamedTypeSymbol ComImportCoClass
         }
 
         #endregion
+
+        #region SkipLocalsInitAttribute
+        private bool _hasSkipLocalsInitAttribute;
+        public bool HasSkipLocalsInitAttribute
+        {
+            get
+            {
+                VerifySealed(expected: true);
```

Comment:
>VerifySealed(expected: true); [](start = 16, length = 29)

While you are in this file, consider adding similar line into the getter above (for ComImportCoClass).